src/App.js

```diff
@@ -10,40 +15,165 @@ class App extends React.Component {
+    let winner = "Both player";
+    if (this.state.player1.score > this.state.player2.score) {
+      winner = "Player 1";
+      this.state.player1.winGame += 1;
+      this.setState({ player1: this.state.player1 });
+    } else if (this.state.player1.score < this.state.player2.score) {
+      winner = "Player 2";
+      this.state.player2.winGame += 1;
+      this.setState({ player2: this.state.player2 });
+    }
+    const restartElem = (
+      <div>
+        <p>{winner} is the final winner! Do you want to play again?</p>
+        <button onClick={this.exeRestart}>Restart</button>
+        <button onClick={window.close}>Quit</button>
       </div>
-    ));
+    );
+
+    this.setState({ restart: restartElem, dealButton: "" });
+  };
+
+  render() {
+    // You can write JavaScript here, just don't try and set your state
+    // You can access your current components state here, as indicated below
+    const currCardElems =
+      "name" in this.state.player1.currCard ? (
+        <Container className="player-table">
+          <Row className="player-table">
+            <Col className="player-table">Player 1</Col>
+            <Col className="player-table">
+              <PlayerCard
+                name={this.state.player1.currCard.name}
+                suit={this.state.player1.currCard.suit}
+              />
+            </Col>
+            <Col className="player-table">{this.state.player1.result}</Col>
+            <Col className="player-table">Score:{this.state.player1.score}</Col>
+          </Row>
+          <Row className="player-table">
+            <Col className="player-table">Player 2</Col>
+            <Col className="player-table">
+              <PlayerCard
+                name={this.state.player2.currCard.name}
+                suit={this.state.player2.currCard.suit}
+              />
+            </Col>
+            <Col className="player-table">{this.state.player2.result}</Col>
+            <Col className="player-table">Score:{this.state.player2.score}</Col>
+          </Row>
+        </Container>
+      ) : (
+        ""
+      );
+
+    const totalWinElem =
+      (this.state.player1.winGame === 0 && this.state.player2.winGame) === 0 ? (
+        ""
+      ) : (
+        <Container className="leaderboard">
+          <Row className="player-table">
+            <center>LeaderBoard</center>
+          </Row>
+          <Row className="player-table">
+            <Col>Player 1</Col>
+            <Col>{this.state.player1.winGame}</Col>
+          </Row>
+          <Row className="player-table">
+            <Col>Player 2</Col>
+            <Col>{this.state.player2.winGame}</Col>
+          </Row>
+        </Container>
+      );
 
     return (
       <div className="App">
         <header className="App-header">
           <h3>High Card 🚀</h3>
+          {totalWinElem}
+          <p>
+            {this.state.round === 0
+              ? ""
+              : "Round Left: " + (26 - this.state.round)}
+          </p>
```

Comment:
```suggestion
          {this.state.round && <p>{"Round Left: " + (26 - this.state.round)}</p>}
```

Reply:
No need to render the p tag for an empty string. Shorthand let's us easily make the conditional rendering